guides/v4.5.0/components/helper-functions.md

```diff
@@ -62,11 +62,43 @@ Since the title is just the `@username` plus some extra stuff, we can replace `@
 
 However, to get the first initial of the string, we'll need to use JavaScript. To do that, we'll write a helper function.
 
-## Writing a Helper Function
+In this case we want a helper function that takes three arguments: a string, a starting position, and a length. The function will return a substring of the original string.
 
-We define helper functions in the `app/helpers` folder.
+## Local Helper Functions
 
-In this case we want a helper function that takes three arguments: a string, a starting position, and a length. The function will return a substring of the original string.
+It's possible to use plain functions for helpers and modifiers.
+
+```js {data-filename="app/components/message.js"}
```

Comment:
I am personally not the biggest fan of using the same example already used for global helper functions but I left it there to discuss. The `substring` helper example that was there initially feels a little bit abstract, especially for local helpers.

Happy to have some input on this before continuing :)

Reply:
I agree that substring feels abstract for local helpers. If you can think of an example that would be similar to a real world use and understandable by junior developers, feel free to replace it. It is also ok to keep using this same example for now, in the interest of keeping things moving with the rest of the refactor. All the real world examples I could think of were complex - filtering over data, displaying dates, and too much math.

Reply:
Sounds good, let me focus on the refactor first before changing the example